zScanner/ViewControllers/NewDocument/NewDocumentPhotosViewController.swift

```diff
@@ -52,7 +52,11 @@ class NewDocumentPhotosViewController: BaseViewController {
     let bottomGradientOverlayHeight: CGFloat = 80
     
     @objc private func takeNewPicture() {
-        showActionSheet()
+        #if targetEnvironment(simulator)
```

Comment:
Better to use the condition `UIImagePickerController.isSourceTypeAvailable(.camera)` 
Sorry If I told you to make condition directly for simulator, We just need to be able to add images on simulator and this condition should do it.

Reply:
No problem. I can do it but it already works. We are able to add images from gallery on simulator.

Reply:
It’s better to use the swift code then compiler flags. 

And it covers edge cases better.